docs/getting-started-guides/rkt/index.md

```diff
@@ -70,110 +65,147 @@ $ cat <<EOF >/etc/rkt/net.d/k8s_flannel_example.conf
+$ export RKT_STAGE1_IMAGE=<stage1-name>
 ```
 
-Then we can launch the local cluster using the script:
+Now you can launch the cluster using the `local-up-cluster.sh` script:
 
 ```shell
 $ hack/local-up-cluster.sh
 ```
 
-We are also working on setting up rkt as the container runtime for [minikube](https://github.com/kubernetes/minikube/issues/168).
+We are also working on getting rkt working as the container runtime in [minikube](https://github.com/kubernetes/minikube/issues/168).
+
+### Launch a rktnetes cluster on Google Compute Engine (GCE)
 
-### Launch a CoreOS/rkt cluster on Google Compute Engine (GCE)
+This section outlines using the `kube-up` script to launch a CoreOS/rkt cluster on GCE.
 
-Here we provide instruction on how to use the `kube-up` script to launch a CoreOS/rkt cluster on GCE.
-In order to do that, you need to specify the OS distribution, project, image:
+First, specify the OS distribution, your GCE project id, and the instance images for the master and nodes. Set the `KUBE_CONTAINER_RUNTIME` to `rkt`:
 
 ```shell
 $ export KUBE_OS_DISTRIBUTION=coreos
-$ export KUBE_GCE_MASTER_PROJECT=coreos-cloud
+$ export KUBE_GCE_MASTER_PROJECT=<my-proj-id>
 $ export KUBE_GCE_MASTER_IMAGE=<image_id>
-$ export KUBE_GCE_NODE_PROJECT=coreos-cloud
+$ export KUBE_GCE_NODE_PROJECT=<my-proj-id>
 $ export KUBE_GCE_NODE_IMAGE=<image_id>
 $ export KUBE_CONTAINER_RUNTIME=rkt
 ```
 
-You can optionally choose the version of rkt used by setting `KUBE_RKT_VERSION`:
+Optionally, set the version of rkt by setting `KUBE_RKT_VERSION`:
 
 ```shell
 $ export KUBE_RKT_VERSION=1.9.1
 ```
 
-Then you can launch the cluster by:
+Optionally, select an alternative [stage1 isolator](#modular-isolation-with-interchangeable-stage1) for the container runtime by setting `KUBE_RKT_STAGE1_IMAGE`:
 
 ```shell
-$ cluster/kube-up.sh
+$ export KUBE_RKT_STAGE1_IMAGE=<stage1-name>
 ```
 
-### Launch a CoreOS/rkt cluster on AWS
+Then you can launch the cluster with:
 
-`kube-up` for AWS is currently unsupported.
-Instead, we recommend you to refer the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS/rkt cluster on AWS.
+```shell
+$ cluster/kube-up.sh
+```
 
-### Deploy apps to your cluster
+### Launch a rktnetes cluster on AWS
+
+The `kube-up` script is not yet supported on AWS. Instead, we recommend following the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS Kubernetes cluster on AWS, then setting kubelet options as above.
+
+### Deploy apps to the cluster
+
+After creating the cluster, you can start deploying applications. For an introductory example, [deploy a simple nginx web server](/docs/user-guide/simple-nginx). Note that this example did not have to be modified for use with a "rktnetes" cluster. More examples can be found in the [Kubernetes examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/).
```

Comment:
I think these links will be expanded in production, see https://github.com/kubernetes/kubernetes.github.io/pull/741#discussion_r68673987 above